### DISCUSSION TITLE
Does this package support backup of dynamic fields

### DISCUSSION THREAD

**Discussion by lengjunbin-trec** [spatie/laravel-backup#1770] [Q&A]
I have a  virtual column that dynamically generated。
when I try to import the database via backup sql.it show me error
`ERROR 3105 (HY000) at line 1828: The value specified for generated column 'RemoteUserIdMakerHashed_2' in table 'Users' is not allowed.`

this field create script is 
`RemoteUserIdMakerHashed json NOT NULL DEFAULT (json_object()),`
`RemoteUserIdMakerHashed_2 varchar(255) COLLATE utf8mb4_unicode_ci GENERATED ALWAYS AS (json_unquote(json_extract(`RemoteUserIdMakerHashed`,_utf8mb4'$."2"'))) VIRTUAL,`

**Comment by MAbughrara** [comment]
any solution in this ? how to exclude VIRTUAL/STORED generated columns?
